components/st-microelectronics/stm-api.stanza

```diff
@@ -130,62 +137,72 @@ defn match-pin? (pin-ref:Ref power:String) :
   if p-ref == power :
     true
 
-defn find-pins-list (obj:JITXObject my-pin:String) :
+; return list of pins that match name (exactly) / don't check for empty
+defn get-pin-list (obj:JITXObject m-pin:String) :
   val result = to-tuple $ for p in pins(obj) filter :
-    if substring?(to-string(ref(p)), my-pin) :
-      match-pin?(ref(p), my-pin)
+    if substring?(to-string(ref(p)), m-pin) :
+      match-pin?(ref(p), m-pin)
   result
 
-defn find-pins? (obj:JITXObject my-pin:String) :
-  val result = to-tuple $ for p in pins(obj) filter :
-    if substring?(to-string(ref(p)), my-pin) :
-      match-pin?(ref(p), my-pin)
+; find the pin exact match return a boolean
+defn find-pin? (obj:JITXObject my-pin:String) :
+  val result = get-pin-list(obj, my-pin)
   not empty?(result)
 
-defn power-pins (stm:JITXObject vdd-pin:PowerPinNames) -> Tuple<JITXObject> :
-  val found-pins = find-pins-list(stm, to-string(vdd-pin))
+defn power-pins (stm:JITXObject vdd-pin:PwrPinNames) -> Tuple<JITXObject> :
+  val found-pins = get-pin-list(stm, to-string(vdd-pin))
```

Comment:
Can we add some context to the error message? This can help us when we debug: 

```
fatal("%_ is missing an identifiable power pin (%_)" % [ref(stm), vdd-pin)])
```